scripts/rotate-aws-creds/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+
+ENVIRONMENT=${1:-}
+CLUSTER_NAME=${2:-}
+if [[ ${ENVIRONMENT} == "" || ${CLUSTER_NAME} == "" ]]; then
+    echo "Usage: $0 <environment> <cluster name> "
+    exit 1
+fi
+
+CLUSTER_ID=$(ocm list clusters | grep "${CLUSTER_NAME}" | awk '{print $1}')
```

Comment:
Cluster names are not unique, let's use ID as the argument.  Also use `--managed` argument to list only OSD clusters.  We can switch arg 2 to be CLUSTER_ID and support both internal ID and EXTERNAL_ID with:
```suggestion
CLUSTER_ID=$(ocm list clusters --managed --columns id,external_id | grep "${CLUSTER_ID}" | awk '{print $1}')
```